src/components/common/EntityInfo.vue

```diff
@@ -130,6 +130,8 @@ export default class EntityInfo extends Mixins(DateMixin) {
 
     // name comes from different places depending on filing type
     switch (this.getFilingType) {
+      case FilingTypes.AMALGAMATION:
+        return (this.getNameRequestApprovedName || numberedDescription)
```

Comment:
Example:

![image](https://github.com/bcgov/business-create-ui/assets/10002889/92eb6173-b47a-4218-a481-79d1924e9098)

This should work fine for an amalgamation with a NR, but I haven't created such a draft for testing yet. I will look into this later.